server/src/http/feature_refresher.rs

```diff
@@ -1245,6 +1267,7 @@ mod tests {
             engine_cache: Arc::new(Default::default()),
             refresh_interval: chrono::Duration::seconds(0),
             persistence: None,
+            open: Default::default(),
```

Comment:
For tests I'm an even bigger fan of explicit

```suggestion
            open: false,
```